components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -120,6 +120,9 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   // Return the parameter list
   const ekat::ParameterList& get_params () const { return m_params; }
 
+  // Return non-const parameter list
+  ekat::ParameterList& get_params_non_const () { return m_params; }
```

Comment:
I didn't like adding this function, but the first time I ask about `enable_column_conservation_checks` for some processes (those that don't set any boundary fluxes) is in `atmosphere_process_group.cpp` where I need to get the parameters from a different process, and need it to have a default value. Let me know if there is a better way to do this.

Reply:
See comment on the line where you use it.